api/endpoints/upload_route.py

```diff
@@ -83,8 +83,8 @@ async def upload_single(
     """
     try:
         file_content = await file.read()
-        logger.info(f"Hash matches: {calculate_md5_checksum(file_content, file_hash)}")
-
+        # Disabled hash check for now
+        # logger.info(f"Hash matches: {calculate_md5_checksum(file_content,file_hash)}")
```

Comment:
I'd be inclined to remove this, unless you anticipate it being used anytime soon?